internal/confmapprovider/discovery/bundle/bundle.d/receivers/smartagent-collectd-kafka-broker.discovery.yaml

```diff
@@ -0,0 +1,47 @@
+#####################################################################################
+#                               Do not edit manually!                               #
+# All changes must be made to associated .tmpl file before running 'make bundle.d'. #
+#####################################################################################
+smartagent/collectd/kafka_broker:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)kafka.*broker"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)kafka.*broker"
```

Comment:
we'll probably want the `not (command matches "splunk.discovery")` filter here, but for maintainability I think we should be dynamically including to to remove the requirement (will look into this).